pennylane/labs/resource_estimation/resource_tracking.py

```diff
@@ -191,12 +194,32 @@ def _counts_from_compressed_res_op(
         _counts_from_compressed_res_op(
             sub_cp_rep, gate_counts_dict, scalar=scalar * counts, gate_set=gate_set, config=config
         )
+
     return
 
 
-def _temp_map_func(op: Operation) -> ResourceOperator:
-    """Temp map function"""
-    raise NotImplementedError
+def _op_to_resource_op(op: Operation) -> ResourceOperator:
```

Comment:
Do you know if this queues the newly created operation? We want to make sure we are not accidentally queue multiple copies of an operation. If it does, we should wrap this function using the stop recording decorator: 

```
@qml.QueuingManager.stop_recording()
def _op_to_resource_op(op): 
    ...
```